src/ui/widgets/Input/input.tsx

```diff
@@ -130,7 +172,7 @@ export const SmartInputComponent = (
     <TextField
       disabled={!enabled}
       value={inputValue}
-      multiline={multiLine}
```

Comment:
The `multiLine` property should still be set here (alongside `maxRows`) otherwise we lose the ability to write across multiple lines.